graphql/schemas/Ecosystem/Filesystem/mapper.graphql

```diff
@@ -32,11 +32,28 @@ type FileSystemImport {
     results: Mixed
 }
 
+input UpdateFilesystemImportInput {
+    mapper_id: ID!
+    name: String!
+    system_module_id: ID!
```

Comment:
@FredPeal why do we need this , its just a update. No need to change system module. If they have to crea a new one